features/step_definitions/admin_steps.rb

```diff
@@ -399,4 +399,10 @@ def update_settings(settings)
   step %{I post the comment "I loved this" on the work "New Work" as a guest with email "#{email}"}
   step %{I should not see "has been blocked at the owner's request"}
   step %{I should see "Comment created!"}
-end
\ No newline at end of file
+end
+
+Then(/^I fill in "([^"]*)" with "([^"]*)'s" invite code$/) do |field, login|
```

Comment:
Sorry, I thought I changed this -- it should be "When" because we're taking an action. /o\